### PR TITLE
kubelet: migrate pkg/kubelet/cloudresource to structured logging

### PR DESCRIPTION
**Pull request by sladyn98**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it:**

structured logging migration for kubelet：
pkg/kubelet/cloudresource/cloud_request_manager.go

**Which issue(s) this PR fixes:**

Ref #98976

Special notes for your reviewer:

**Does this PR introduce a user-facing change?**
```release-note
migrate pkg/kubelet/cloudresource to structured logging
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:**
```
- [KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
```

### CODE REVIEW

File: pkg/kubelet/cloudresource/cloud_request_manager.go [hunk 1]
```diff
@@ -103,7 +103,7 @@ func (m *cloudResourceSyncManager) getNodeAddresses() ([]v1.NodeAddress, error)
 }
 
 func (m *cloudResourceSyncManager) syncNodeAddresses() {
-	klog.V(5).Infof("Requesting node addresses from cloud provider for node %q", m.nodeName)
+	klog.V(5).InfoS("Requesting node addresses from cloud provider for node","nodeName", m.nodeName)
```
Comment (serathius) [inline pkg/kubelet/cloudresource/cloud_request_manager.go:106]:
```suggestion
	klog.V(5).InfoS("Requesting node addresses from cloud provider for node", "nodeName", m.nodeName)
```

File: pkg/kubelet/cloudresource/cloud_request_manager.go [hunk 2]
```diff
@@ -112,7 +112,7 @@ func (m *cloudResourceSyncManager) syncNodeAddresses() {
 	defer m.nodeAddressesMonitor.Broadcast()
 
 	if err != nil {
-		klog.V(2).Infof("Node addresses from cloud provider for node %q not collected: %v", m.nodeName, err)
+		klog.V(2).InfoS("Node addresses from cloud provider for node not collected","nodeName", m.nodeName, "err", err)
```
Comment (serathius) [inline pkg/kubelet/cloudresource/cloud_request_manager.go:115]:
```suggestion
		klog.V(2).InfoS("Node addresses from cloud provider for node not collected", "nodeName", m.nodeName, "err", err)
```

File: pkg/kubelet/cloudresource/cloud_request_manager.go [hunk 3]
```diff
@@ -124,7 +124,7 @@ func (m *cloudResourceSyncManager) syncNodeAddresses() {
 		return
 	}
 
-	klog.V(5).Infof("Node addresses from cloud provider for node %q collected", m.nodeName)
+	klog.V(5).InfoS("Node addresses from cloud provider for node collected","nodeName", m.nodeName)
```
Comment (serathius) [inline pkg/kubelet/cloudresource/cloud_request_manager.go:127]:
```suggestion
	klog.V(5).InfoS("Node addresses from cloud provider for node collected", "nodeName", m.nodeName)
```

**Comment by serathius** [comment]
Minor comments about format, otherwise lgtm. 
/ok-to-test

**Comment by kikisdeliveryservice** [comment]
@sladyn98 could you please reformat the description so the bot can pickup your release note?

**Comment by sladyn98** [comment]
I guess my release not will now be picked up.

**Comment by serathius** [comment]
@sladyn98  Could you clarify what you mean?
Could you rebase your code on master branch to fix bazel test?

**Comment by ehashman** [comment]
@sladyn98 your release note must start with ```release-note. Currently it only has the backticks. Without the accompanying "release-note" text the bot won't see it.

Please rebase your PR rather than merging or pulling changes.

**Comment by ehashman** [comment]
Close! Try quoting my reply and copying it for your release note.

```release-note
migrate pkg/kubelet/cloudresource to structured logging
```

/retest

**Review by ehashman — COMMENTED** [review]
/lgtm

**Pull request merged**